src/UBAFeeCalculator/UBAFeeSpokeCalculatorAnalog.test.ts

```diff
@@ -92,6 +93,16 @@ describe("UBAFeeSpokeCalculatorAnalog", () => {
       const lastIncentiveBalance = BigNumber.from(1000);
       const chainId = 1;
       const flowType = "inflow";
+
+      const zeroFeePoint = defaultConfig.getZeroFeePointOnBalancingFeeCurve(chainId);
+      const feeToBringFeePctToZero = computePiecewiseLinearFunction(
+        defaultConfig.getBalancingFeeTuples(chainId),
+        zeroFeePoint,
+        lastRunningBalance
+      ).abs();
+
+      expect(feeToBringFeePctToZero.gt(lastIncentiveBalance)).toBeTruthy();
```

Comment:
why not use `expect().to.be.greater.than()`? 

Also, can you print out what you think the non discounted balancing fee and what the discounted one should be? I think we should check that the discount factor is computed and then applied correctly.

For example if we expect the discount factor to be 0.8, we should check that balancingFee.eq(potentialBalancingFee.mul(toBNWei("0.8")).div(fixedPointAdjustment))

Reply:
These are `jest` tests & they don't compute the inequalities on BigNumbers out of the box